packages/rocketchat-authorization/client/views/permissionsRole.html

```diff
@@ -21,6 +21,11 @@
 						<option value="Users" selected="{{$eq scope 'Users'}}">{{_ "Global"}}</option>
 						<option value="Subscriptions" selected="{{$eq scope 'Subscriptions'}}">{{_ "Rooms"}}</option>
 					</select>
+
+					<br/>
+					<label for="mandatory2fa">{{_ "Users must use Two Factor Authentication"}} :</label>
```

Comment:
we have not created a translation key for `Users must use Two Factor Authentication`

Reply:
Added.